src/System.ServiceModel.Syndication/tests/BasicScenarioTests.cs

```diff
@@ -368,43 +358,130 @@ public static void AtomEntryPositiveTest_write()
         }
 
         [Fact]
-        [ActiveIssue(25058)]
-        public static void AtomFeedPositiveTest_WsFeedIcon()
+        public static void AtomFeedPositiveTest()
         {
-            string file = "ws-feed-icon.xml";
-            AtomFeedPositiveTest(file);
+            string dataFile = @"atom_feeds.dat";
+            List<string> fileList = GetTestFilesForFeedTest(dataFile);
+            List<AllowableDifference> allowableDifferences = GetAtomFeedPositiveTestAllowableDifferences();
+
+            foreach (string file in fileList)
+            {
+                FeedPositiveTest(file, FeedFormatterType.Atom10FeedFormatter, allowableDifferences);
+            }
         }
 
         [Fact]
-        public static void AtomFeedPositiveTest()
+        public static void RssEntryPositiveTest()
         {
-            string dataFile = @"atom_feeds.dat";
+            string file = @"RssEntry.xml";
+            EntryPositiveTest(file, SyndicationItemFormatterType.Rss20ItemFormatter);
+        }
+
+        [Fact]
+        public static void RssFeedPositiveTest()
+        {
+            string dataFile = @"rss_feeds.dat";
             List<string> fileList = GetTestFilesForFeedTest(dataFile);
+            List<AllowableDifference> allowableDifferences = GetRssFeedPositiveTestAllowableDifferences();
 
             foreach (string file in fileList)
             {
-                AtomFeedPositiveTest(file);
+                FeedPositiveTest(file, FeedFormatterType.Rss20FeedFormatter, allowableDifferences);
             }
         }
 
-        private static void AtomFeedPositiveTest(string file)
+        private static void EntryPositiveTest(string file, SyndicationItemFormatterType syndicationItemFormatterType)
```

Comment:
Nit: Can you please rename the method to "ReadWriteSyndicationItem"?

Reply:
Done.